recipes-support/fastrpc/fastrpc/mount-dsp.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+#
+# Copyright (c) 2021 Linaro
```

Comment:
2021-2024 or 2021,2023,2024

Reply:
My understanding (I'm not a lawyer) is that the year of first publication is sufficient, which is why I added just the first one. I also saw legal discussions saying that the year is not really needed in the end, but I can update to cover the range (as it is also common based on different projects).

Reply:
I know that Qualcomm engineers frequently change (c) years when doing the contributions. So do GNU projects.

Reply:
Yup, the need for not having to update the years is a recent understanding (end of last year).

Reply:
Added the range.